creativecloud/blocks/interactive-marquee/interactive-marquee.js

```diff
@@ -83,7 +83,7 @@ function interactiveInit(el, decorateButtons, decorateBlockBg, createTag) {
 export default async function init(el) {
   const miloLibs = getLibs('/libs');
   const { decorateButtons, decorateBlockBg } = await import(`${miloLibs}/utils/decorate.js`);
-  const { createTag, loadStyle } = await import(`${miloLibs}/utils/utils.js`);
+  const { createTag, loadStyle, createIntersectionObserver } = await import(`${miloLibs}/utils/utils.js`);
```

Comment:
Can we now import this globally from our utils? It is exported there. We can add createIntersectionObserver to the list

https://github.com/adobecom/cc/blob/stage/creativecloud/scripts/utils.js